src/components/Pages/Cart/Cart.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import {Box, Typography} from "@material-ui/core";
+
+import './Cart.scss'
+import CartItemCard from "./CartItemCard/CartItemCard";
+import CartPrice from "./CartPrice/CartPrice";
+
+const database = [
+  {
+    "id": 0,
+    "img": {
+      "path": "/images/bouquets/img1.jpg",
+      "title": "Рожева гіпсофіла"
+    },
+    "price": 450
+  },
+  {
+    "id": 1,
+    "img": {
+      "path": "/images/bouquets/img2.jpg",
+      "title": "Рожеві тюльпани"
+    },
+    "price": 500
+  },
+  {
+    "id": 2,
+    "img": {
+      "path": "/images/bouquets/img3.jpg",
+      "title": "Кущові троянди",
+      "height": "300"
+    },
+    "price": 450
+  },
+];
+
+// TODO: change it to dynamically select products rather than hard-code)
+// TODO: probably use here Redux or sth else
+const dataSelected = [
+  {
+    id: 0,
+    quantity: 1,
+  },
+  {
+    id: 1,
+    quantity: 2,
+  },
+  {
+    id: 2,
+    quantity: 1,
+  }
+];
+
+export default class Cart extends React.Component {
+  /**
+   * products: is an Array of {
+   *   id:        Int, // id of product in database
+   *   quantity:  Int, // the quantity of the product selected by user
+   * }
+   *
+   * @type {{totalPrice: number, products: []}}
+   */
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      products: dataSelected || [],
+      totalPrice: 0
+    };
+
+    this.freeDeliveryFrom = 500;
+    this.deliveryCost = 100;
+  }
+
+  getItemById(id) {
+    return database[id];
+  }
+
+  calculateTotalPrice() {
+    let totalPrice = 0;
+
+    this.state.products.forEach(
```

Comment:
use reduce instead of `forEach`

Reply:
`reduce` takes the first element as an accumulator. So it will be needed to call sth like `.map().reduce()`, but it's unnecessary O(n) memory consumption